packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -210,6 +215,7 @@ const ApplicationNavigation = ({
           returnFocusOnDeactivate: false,
         }}
         className={cx('drawer-menu-focus-trap-container')}
+        id={id}
```

Comment:
I don't think I'd expect the default id to show up on the FocusTrap element. This would likely be better placed on the outer container on line 447.

Reply:
Yup, you're right. I tested with just removing that completely and it's actually not needed at all.